docs/workers.md

```diff
@@ -296,6 +296,34 @@ have different characteristics and so admins
 may wish to run multiple groups of workers handling different endpoints so that
 load balancing can be done in different ways.
 
+You could create multiple worker configuration files manually or alternatively use 
+a simple `bash` script. For example:
+
+```#!/bin/bash
+for i in {1..5}
+do
+cat << EOF >> generic_worker$i.yaml
+worker_app: synapse.app.generic_worker
+worker_name: generic_worker$i
+
+# The replication listener on the main synapse process.
+worker_replication_host: 127.0.0.1
+worker_replication_http_port: 9093
+
+worker_listeners:
+  - type: http
+    port: 808$i
+    resources:
+      - names: [client, federation]
+
+worker_log_config: /etc/matrix-synapse/generic-worker-log.yaml
+EOF
+done
+```
+
+This would create five generic workers with a unique `worker_name` field
+in each file and listening on ports 8081-8085.
+
```

Comment:
this isn't the right place in this file - rather it would need to be in the "Worker configuration" section (https://matrix-org.github.io/synapse/latest/workers.html#worker-configuration).

However, I don't think that's a great place for it either. We're concerned that the docs are not a great place to be storing a script of this size (with the opportunity to grow in the future as people wish for it to do more things), and it's only one piece in the puzzle of setting up workers.

Still, it does look potentially useful. How about putting it in a directory in the `contrib` directory, and we can link to it from the "Worker configuration" section?